addons/survey/views/survey_templates_statistics.xml

```diff
@@ -70,70 +63,51 @@
         <t t-set="graph_data" t-value="question_data['graph_data']"/>
         <t t-set="table_data" t-value="question_data['table_data']"/>
 
-        <div>
-        <div class="d-flex align-items-start mb-2">
-            <div class="mr-auto">
-                <h5 t-field="question.title" class="mb-1"/>
-                <div t-if="question_scoring">
-                    <t t-if="question.question_type in ['simple_choice', 'multiple_choice']">
-                        <t t-if="len(question_data['right_answers']) >= 1">
-                            <span t-if="len(question_data['right_answers']) == 1">Right answer: </span>
-                            <span t-if="len(question_data['right_answers']) > 1">Right answers: </span>
-                            <strong>
-                                <t t-foreach="question_data['right_answers']" t-as="answer">
-                                    <span t-if="answer_index != 0">, </span><span t-esc="answer.value"/>
-                                </t>
-                            </strong>
-                        </t>
-                    </t>
-                    <t t-elif="question.is_scored_question">
-                        <span>Right answer:</span>
-                        <strong>
-                            <t t-if="question.question_type == 'numerical_box'">
-                                <span t-esc="question.answer_numerical_box"/>
-                            </t>
-                            <t t-if="question.question_type == 'date'">
-                                <span t-field="question.answer_date"/>
-                            </t>
-                            <t t-if="question.question_type == 'datetime'">
-                                <span t-field="question.answer_datetime"/>
-                            </t>
-                        </strong><br/>
-                        <span>Score:</span>
-                        <strong>
-                            <span t-esc="question.answer_score"/>
-                        </strong>
-                    </t>
+        <div class="survey_question pb-5 border-bottom">
+            <div class="d-flex align-items-start mb-2">
+                <div class="mr-auto">
+                    <h5 t-field="question.title" class="mb-3"/>
                 </div>
-            </div>
-            <!-- Question info -->
-            <span class="badge badge-info" t-field='question.question_type'/>
-            <t t-if="question.question_type == 'matrix'">
-                <span class="badge badge-info ml-2" t-field='question.matrix_subtype'/>
-            </t>
-            <!-- Scoring info -->
-            <t t-if="question_scoring">
```

Comment:
Could we remove the "question_scoring" variable completely?

It looks like "is_scored_question" is just better.